api/node/node_service.yaml

```diff
@@ -223,4 +223,32 @@ paths:
                 format: binary
         "204":
           description: did not find a message to retrieve
+  /proposal/{layer}/{node}:
+    get:
+      summary: Get a partial proposal for a given node in a layer
+      tags:
+        - "proposals"
+      parameters:
+        - in: path
+          name: layer
+          required: true
+          schema:
+            $ref: "models/components.yaml#/components/schemas/LayerID"
+        - in: path
+          name: node
+          required: true
+          schema:
+            $ref: "models/components.yaml#/components/schemas/NodeID"
+      responses:
+        "200":
+          description: successfully created a partial proposal
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
```

Comment:
It's true - the response of this endpoint cannot be efficiently cached as it will be different for every client... However the same isn't true for few other endpoints. For example, `/hare/total_weight/{layer}` returns a weight (integer) in a binary form which seems to be an overkill. Let's leave it this way for now and refine the interface when it's more clear what is needed and what not.